warehouse/accounts/views.py

```diff
@@ -456,7 +547,6 @@ def _login_user(request, userid):
 
     # Whenever we log in the user, we want to update their user so that it
     # records when the last login was.
-    user_service = request.find_service(IUserService, context=None)
```

Comment:
Is there a point to this change? It seems kind of churny for churn sake, but I might just be missing it.

Reply:
I don't think there is; it was part of the original branch's changeset. I'll test and see what, if anything, breaks.